aiohttp/helpers.py

```diff
@@ -714,6 +720,18 @@ def _parse_content_type(self, raw):
         else:
             self._content_type, self._content_dict = cgi.parse_header(raw)
 
+    def _parse_content_disposition(self, _CONTENT_DISPOSITION):
+        from . import multipart
+        raw = self._headers.get(_CONTENT_DISPOSITION)
+        if raw is None:
+            self._content_disposition = None
+        else:
+            value, params = multipart.parse_content_disposition(raw)
+            filename = multipart.content_disposition_filename(params)
+            self._content_disposition = ContentDisposition(value, params,
```

Comment:
Convert `params` into `types.MappingProxyType`. The value should be immutable

Reply:
Will do in next patchset.